stories/2021-09-08-Desert-Island-Disks/Desert-Island-Discs.ipynb

```diff
@@ -0,0 +1,2839 @@
+{
```

Comment:
<p>Could you better explain the concept of centrality and why this interpretation was come to here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/165/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='79'/>

Reply:
Addressed in [d5e78cf](https://github.com/alan-turing-institute/TuringDataStories/pull/165/commits/d5e78cf0f0389ae14d548b405b98418fbe013c07)